scripts/webpack/webpack.dev.babel.js

```diff
@@ -55,7 +55,7 @@ module.exports = (env) => webpackConfigBase({
         + "style-src 'self' 'unsafe-inline' https://code.s4d.io; "
         + "media-src 'self' https://code.s4d.io https://*.clouddrive.com https://*.giphy.com https://*.webexcontent.com data: blob:; "
         + "font-src 'self' https://code.s4d.io; "
-        + "img-src 'self' https://*.clouddrive.com https://code.s4d.io https://*.webexcontent.com data: blob: https://*.rackcdn.com; "
+        + "img-src 'self' https://*.clouddrive.com https://code.s4d.io https://*.webexcontent.com data: blob: https://*.rackcdn.com https://cisco.webex.com; "
```

Comment:
Without this the img tag was not fetching the captcha from the url.

Reply:
@Shreyas281299 what would happen in INT environment?

Reply:
Im unable to find any integration setup in the repo. All the urls being used are prod . I tried using an int access-token to register but its failing as it is communicating with the prod urls. I think its safe to assume that there is no way to use space-widgets in int env.